### DISCUSSION TITLE
It says there are "outstanding issues" with my account when applying for a Students account

### DISCUSSION THREAD

**Discussion by AnonymousMister** [community/community#63365] [GitHub Education]
I finally decided to apply for GitHub students account today. However, when I tried sending my application I was immediately greeted with this message:
“There are outstanding issues with your GitHub account. Please contact [GitHub Support](https://support.github.com/contact) for more information.”
I don't know how to solve it this problem

**Comment by arjunkumarbose** [comment]
https://support.github.com/request
under Global Campus verification help^ section
Go threw your issue here, GitHub support will solve the issue immediately :D

**Reply by AnonymousMister** [reply]
Thank you

**Reply by arjunkumarbose** [reply]
You're welcome! If this solves your issue, mark it as the answer so that if anyone faces the same issue, he might find it useful 😊

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @AnonymousMister , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
